js/storage.md

```diff
@@ -61,13 +61,16 @@ Amplify.configure({
         userPoolWebClientId: 'XX-XXXX-X_abcd1234', //OPTIONAL - Amazon Cognito Web Client ID
     },
     Storage: {
-        bucket: '', //REQUIRED -  Amazon S3 bucket
-        region: 'XX-XXXX-X', //OPTIONAL -  Amazon service region
+        AWSS3: {
```

Comment:
You will need to update line 157 `Storage.configure`

Reply:
done!